ios/TabViewImpl.swift

```diff
@@ -75,13 +78,14 @@ struct TabViewImpl: View {
       if #available(iOS 15.0, *) {
         UITabBar.appearance().scrollEdgeAppearance = configureAppearance(for: newValue ?? "")
       }
+      UITabBar.appearance().standardAppearance = configureAppearance(for: newValue ?? "")
```

Comment:
It would be best if you created a separate onChange() because this reacts only to `props.scrollEdgeAppearance`

It would be best if you had separate onChange for each prop